apps/admin/sbox-intsvc/base/kustomization.yaml

```diff
@@ -3,7 +3,12 @@ kind: Kustomization
 resources:
   - ../../base
   - kube-slack-values.yaml
-  - ../../csi-secret-store-provider-v1.0.1
-  - traefik-values.yaml
   - ../../../rbac/reader-clusterrole.yaml
-  - ../../traefik/traefik.yaml
+  - ../../secrets-csi-driver-crds
+  - ../../secrets-csi-driver
+  - ../../traefik2
+  - ../../aad-pod-id/mi
+
+patchesStrategicMerge:
+  - ../../traefik2/sbox/secret-provider.yaml
```

Comment:
This isn't correct?

Reply:
Corrected this, added own sbox-intsvc secret provider file and granted access PTLSBOX MI to that access policy.